### PR TITLE
Feature: Merge add/create diagram buttons

### LINKED ISSUE #471: Merge "Add a diagram" and "create a diagram" button into one single button list
## Description

The buttons "Add a diagram" and "create a diagram" are not really easy to understand.
The purpose of this task is to merge the 2 buttons into a single button list.
But the behavior should stay the name.

### PR DESCRIPTION
**Pull request by aboubakar14**
resolve #471

### CODE REVIEW

File: src/components/card/DiagramsCard.vue [hunk 1]
```diff
@@ -11,37 +11,49 @@
         <q-btn
           outline
           no-caps
-          class="q-ml-xl"
+          label="Create diagram"
```
Comment (justinehell) [inline src/components/card/DiagramsCard.vue:14]: use i18n

File: src/i18n/en-US/index.js [hunk 2]
```diff
@@ -37,16 +37,16 @@ export default {
...
         button: {
-          name: 'Create a diagram',
-          title: 'Open dialog to create new model',
+          scratch: {
+            name: 'Create a diagram from scratch',
+            title: 'Open a popup to create a new diagram',
+          },
+          template: {
+            name: 'Create a diagram from a template',
+            title: 'Open a popup to create a diagram from a template',
```
Comment (k-y0u) [inline src/i18n/en-US/index.js:48]: is it not a "new diagram" like above ?

File: src/components/card/DiagramsCard.vue [hunk 3]
```diff
@@ -11,37 +11,49 @@
...
+                  ? $t('actions.models.create.button.scratch.title')
+                  : $t('errors.permissionsDenied')"
+                data-cy="create-diagram-from-scratch-button"
+                :disable="!$acl.role('create-diagram')"
+                @click="DialogEvent.next({
+                  type: 'open',
+                  key: 'CreateModel',
+                })"
+              >
+                From scratch
```
Comment (justinehell) [inline src/components/card/DiagramsCard.vue:36]: use i18n

File: src/components/card/DiagramsCard.vue [hunk 4]
```diff
@@ -11,37 +11,49 @@
...
+                  ? $t('actions.models.create.button.template.title')
+                  : $t('errors.permissionsDenied')"
+                data-cy="create-diagram-from-template-button"
+                :disable="!$acl.role('create-diagram-from-template')"
+                @click="DrawerEvent.next({
+                  key: 'CreateDiagram',
+                  type: 'open',
+                })"
+              >
+                From template
```
Comment (justinehell) [inline src/components/card/DiagramsCard.vue:52]: use i18n

File: src/components/card/DiagramsCard.vue [hunk 5]
```diff
@@ -11,37 +11,49 @@
...
+          data-cy="create-diagram-list-button"
+        >
+          <q-menu auto-close>
+            <q-list
+              style="min-width: 100px"
+            >
+              <q-item
+                clickable
+                :label="$t('actions.models.create.button.scratch.name')"
+                :title="$acl.role('create-diagram')
```
Comment (k-y0u) [inline src/components/card/DiagramsCard.vue:26]: better put the ternary out of the template part, no ?

File: src/components/card/DiagramsCard.vue [hunk 6]
```diff
@@ -11,37 +11,49 @@
...
-            ? $t('actions.models.create.button.title')
-            : $t('errors.permissionsDenied')"
-          data-cy="create-model-button"
-          :disable="!$acl.role('create-diagram')"
-          @click="DialogEvent.next({
-            type: 'open',
-            key: 'CreateModel',
-          })"
-        />
+          data-cy="create-diagram-list-button"
```
Comment (justinehell) [inline src/components/card/DiagramsCard.vue:17]: I would just keep `data-cy="create-diagram-button"`

File: src/components/card/DiagramsCard.vue [hunk 7]
```diff
@@ -11,37 +11,49 @@
...
+                  key: 'CreateModel',
+                })"
+              >
+                From scratch
+              </q-item>
+              <q-separator />
+              <q-item
+                clickable
+                :label="$t('actions.models.create.button.template.name')"
+                :title="$acl.role('create-diagram-from-template')
```
Comment (k-y0u) [inline src/components/card/DiagramsCard.vue:42]: better put the ternary out of the template part, no ?

File: src/components/card/DiagramsCard.vue (hunk 1 above)
Comment (Zorin95670) [inline src/components/card/DiagramsCard.vue:14]: translation

File: changelog.md [hunk 8]
```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Improve performance on git add.
 * Replace OIDC authentication by Leto-Modelizer-Api, see [this issue](https://github.com/ditrit/leto-modelizer/issues/425).
 * Separating git-related functions into a dedicated composable.
+* Merge add/create diagram buttons, see [this issue](https://github.com/ditrit/leto-modelizer/issues/471).
```
Comment (k-y0u) [inline changelog.md:39]: ... into a drop down button, see...

**Review by justinehell — CHANGES_REQUESTED** [review]

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/card/DiagramsCard.vue (hunk 3 above)
Comment (Zorin95670) [inline src/components/card/DiagramsCard.vue:36]: translation

File: src/components/card/DiagramsCard.vue (hunk 4 above)
Comment (Zorin95670) [inline src/components/card/DiagramsCard.vue:52]: translation

File: src/components/card/DiagramsCard.vue (hunk 5 above)
Comment (aboubakar14) [reply src/components/card/DiagramsCard.vue:26]: We did it this way in every other pages.

File: src/components/card/DiagramsCard.vue (hunk 7 above)
Comment (aboubakar14) [reply src/components/card/DiagramsCard.vue:42]: We did it this way in every other pages.

File: src/i18n/en-US/index.js (hunk 2 above)
Comment (aboubakar14) [reply src/i18n/en-US/index.js:48]: Removed in both

File: src/components/card/DiagramsCard.vue (hunk 6 above)
Comment (aboubakar14) [reply src/components/card/DiagramsCard.vue]: done

File: src/components/card/DiagramsCard.vue (hunk 3 above)
Comment (aboubakar14) [reply src/components/card/DiagramsCard.vue]: done

File: src/components/card/DiagramsCard.vue (hunk 1 above)
Comment (aboubakar14) [reply src/components/card/DiagramsCard.vue:14]: done

**Review by justinehell — DISMISSED** [review]

File: changelog.md (hunk 8 above)
Comment (aboubakar14) [reply changelog.md:39]: done

**Review by justinehell — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

File: src/components/card/DiagramsCard.vue (hunk 7 above)
Comment (Zorin95670) [reply src/components/card/DiagramsCard.vue:42]: Agree with @aboubakar14

File: src/components/card/DiagramsCard.vue (hunk 5 above)
Comment (Zorin95670) [reply src/components/card/DiagramsCard.vue:26]: Agree with @aboubakar14

**Review by k-y0u — APPROVED** [review]

**Pull request merged**